docs/API_REFERENCE.md

```
@@ -276,4 +294,4 @@ Project task routes under `/api/v1/projects`:
 - [modules/brand-studio.md](modules/brand-studio.md)
 - [modules/customer-portal.md](modules/customer-portal.md)
 - [modules/ai-estimate-assist.md](modules/ai-estimate-assist.md)
-- [modules/settings-and-operations.md](modules/settings-and-operations.md)
+- [modules/settings-and-operations.md](modules/settings-and-operations.md)
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)